packages/core/src/components/dropdown-menu/dropdown-menu.template.ts

```diff
@@ -16,7 +16,7 @@ export function dropdownMenuTemplate({
       <button type="button" class=${`dso-${button.variant}`} slot="toggle">
         <span>${button.label}</span>
       </button>
-      <div class="dso-dropdown-options">
+      <div class="dso-dropdown-options" role="menu">
```

Comment:
net zoals we de `ul`s en de `li`s in het component verrijken met de nodige attributen, kunnen we dat met deze toch ook wel doen?

Reply:
... zodat het niet meer als onderdeel van de voorgeschreven markup hoeft?

Reply:
Ah, ja, verkeerd gezien, dacht dat het onderdeel was van het component. Aangepast, wordt nu via script gezet.

Reply:
✔